.github/workflows/build_upload_conda.yml

```diff
@@ -0,0 +1,33 @@
+name: Build and Upload Conda Package When Release is Published
+
+# Triggers when a release is made (in our case by the run_deploy_on_merge.yml action)
+on:
+  workflow_dispatch:
+  release:
+    types: [published]
+
+env:
+  ANACONDA_API_TOKEN: ${{ secrets.CONDA_TOKEN }}
+  PHG2_VERSION: ${{ vars.PHG2_VERSION }}
+  PHG2_RELEASE: ${{ vars.PHG2_RELEASE }}
+  PHG2_VERSION_MD5: ${{ vars.PHG2_VERSION_MD5 }}
+
+jobs:
+  build-upload-conda:
```

Comment:
Sure can do!

Reply:
[Let me know what you think](https://github.com/maize-genetics/phg_v2/pull/67/files?short_path=b335630#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5).

We can add more to the Quick Start in future - I wasn't sure how much we wanted included since stuff is still in flux.

Reply:
Here is my concern:  If they pull the apps into the base conda environment, then the programs are seen via ProcessBuilder().  I think this is what you originally tested.  I did not understand if this worked when our Clikt commands precede the ProcessBuilder() commands with a conda environment setting.  Do all environments inherit what is in the base environment?  if yes, then this was ok.

But I think we agreed we don't want to encourage adding to the base environment.


If users create a new conda environment and load into that  this will not work with the ProcessBuilder() commands unless the environment is one that we know.  So either the user should always create a conda environment named phgv2-conda , or they will have to pass a new paramter with the conda environment name to every class that has a ProcessBuilder call.

let me know if I'm missing something here.

Reply:
Please [see here](https://github.com/maize-genetics/phg_v2/pull/67#issuecomment-1850877235) for a response - I wanted to make sure everybody was on the same page and understood the discussion from this morning.